packages/components/src/chart-uplot/lib/get-gradient-fill.ts

```diff
@@ -0,0 +1,42 @@
+import type { ScaleGradientFunction } from '../hooks/use-scale-gradient';
+import type uPlot from 'uplot';
+
+export default function getGradientFill(
```

Comment:
Extracted from the options object declaration in `index.tsx` without modification.